### ISSUE TITLE
why line numbers start with 0?

### ISSUE THREAD

**Issue by rushkii** [bennymi/svhighlight#8]
line numbers should not start with 0.

![image](https://github.com/bennymi/svhighlight/assets/47179236/e1f077d5-b654-4c21-a0fe-609ef8862960)

**Comment by caoccao** [comment]
Yes, I met the same issue. Is it possible to set the line number of the first line?
